### PR TITLE
Remove obsolete script

### PR DESCRIPTION
**Pull request by riuvshin**
As any CHE stack should work on openshift we don't need `replace_stacks.sh` anymore

related to: https://github.com/eclipse/che/pull/6618

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh [hunk 1]
```diff
@@ -12,9 +12,7 @@
 #   curl -fsSL ${DEPLOY_SCRIPT_URL} -o get-che.sh
 #   WAIT_SCRIPT_URL=https://raw.githubusercontent.com/eclipse/che/master/dockerfiles/cli/scripts/openshift/wait_until_che_is_available.sh
 #   curl -fsSL ${WAIT_SCRIPT_URL} -o wait-che.sh
-#   STACKS_SCRIPT_URL=https://raw.githubusercontent.com/eclipse/che/master/dockerfiles/cli/scripts/openshift/replace_stacks.sh
-#   curl -fsSL ${STACKS_SCRIPT_URL} -o stacks-che.sh
-#   bash get-che.sh && wait-che.sh && stacks-che.sh
+#   bash get-che.sh && wait-che.sh
```
Comment (l0rd) [inline dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: I think you can remove script `wait-che.sh` too. WDYT?

**Review by benoitf — COMMENTED** [review]
Up to @l0rd

File: dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh (hunk 1 above)
Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: I think that script still could be useful... Myabe it is better to move it to deploy_che as a func and remove wait CHE funcs from the ocp.sh 
@sleshchenko @l0rd WDYT?

Comment (sleshchenko) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: @riuvshin I like this idea. Do you think it should always wait for Che or make it an option?

Also `get-che.sh` file name makes me a bit confused. Maybe it is better to rename it to `deploy_che.sh` as the original file.

Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: that could be an option for sure, just to not break current approach

Comment (l0rd) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: +1 to add it as a function in `deploy_che.sh`, making it an option may be interesting but also part of another PR. It's up to you.

Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: @sleshchenko @l0rd PR updated

Comment (sleshchenko) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: Should `bash get-che.sh --wait-che` be here?

Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh]: @sleshchenko nice catch, ty, updated.

**Review by garagatyi — APPROVED** [review]

**Review by l0rd — APPROVED** [review]
nice :-)

**Pull request merged**